src/ServicePulse.Host/app/js/services/service.formatter.js

```diff
@@ -33,6 +33,10 @@
             }
         }
 
+        function round(num, decimals) {
+            return +(Math.round(num + `e+${decimals}`) + `e-${decimals}`);
```

Comment:
to answer a question from which dimension comes this code:
https://stackoverflow.com/a/19722641